src/Core/cldj_cmn_mod.F90

```diff
@@ -46,14 +45,14 @@ MODULE CLDJ_CMN_MOD
       ! JVN_ :  max # of J-values
 #ifdef MODEL_GEOSCHEM
       integer, parameter :: JVN_ = 166
-#else
+#elif CLOUDJ_STANDALONE
       integer, parameter :: JVN_ = 101
 #endif
 
       ! mAN_ :  max # FJX aerosols in layer (needs NDX for each)
 #ifdef MODEL_GEOSCHEM
       integer, parameter :: AN_=37
-#else
+#elif CLOUDJ_STANDALONE
```

Comment:
Suggestion: add error traps such as:
```F90
#ifdef MODEL_GEOSCHEM
...
#elif CLOUDJ_STANDALONE
...
#else
#error "Invalid model selection: Must be either "CLOUDJ_STANDALONE" or "MODEL_GEOSCHEM"
#endif
```
This will force a compile-time error.

Reply:
Maybe you only need to add that to the first #if block.

Reply:
Good catch. I want to eventually remove the need for model-dependent blocks but for now an error trap is a good idea.